forge-gui/res/cardsfolder/upcoming/ulamogs_dreadsire.txt

```diff
@@ -0,0 +1,9 @@
+Name:Ulamog's Dreadsire
+ManaCost:10
+Types:Creature Eldrazi
+PT:10/10
+K:Vigilance
+K:Ward:Sac<1/Permanent.cmcGE1>

```

Comment:
The Sac Cost probably needs a better description

Reply:
Wait, I corrected that as a separate commit?